package.json

```diff
@@ -17,11 +17,11 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev",
-    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev-cloud",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
-    "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
+    "start:dev:oss": "yarn start:dev",
+    "start:dev:remocal": "NODE_ENV=development UI_SHA=local_dev_mode BASE_PATH=/ API_BASE_PATH=/ PORT=8080 PUBLIC=${PUBLIC:-https://twodotoh.a.influxcloud.dev.local/} CLOUD_URL=/auth yarn start",
+    "start:kind": "echo 'please start using `yarn start:dev:remocal`' && yarn start:dev:remocal",
     "start:docker": "yarn generate && yarn run start:dev",
```

Comment:
Note: the `yarn start:docker` is the only additional scripts which I wasn't sure if could be deleted. It's refers to part of the influxdb/oss build workflow, and I believe it's legacy (no longer used). We can clean that up later, if needed.